client/stylesheets/_toCat.scss

```diff
@@ -3647,8 +3651,8 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .desc-box p {
-    width: 75%;
-    padding: 10px;
+    width: 95%;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered margin, padding, width